src/amber/controller/base.cr

```diff
@@ -32,5 +32,12 @@ module Amber::Controller
     def session
       context.session
     end
+
+    def halt!(status_code : Int32 = 200, content = "")
```

Comment:
If we're going to use this method to set status_code and content shouldn't it do headers as well?